src/parts/_misc.css

```diff
@@ -148,3 +148,150 @@ footer {
 body > footer {
   margin-top: 40px;
 }
+
+progress {
+  appearance: none;
+  height: 8px;
+  position: relative;
+  width: 100%;
+}
+
+/* Container (background) of the progress bar */
+progress[value] {
+  border: 1px solid var(--border);
+  border-radius: 6px;
+  background-color: var(--border);
+
+  /* for IE 10+ */
+  color: var(--focus);
+}
+
+/* IE provides it's own animation which looks nice here */
+progress:not([value]) {
+  background-color: var(--border);
+  border-radius: 6px;
+  border: none;
+}
+
+::-webkit-progress-bar {
+  background-color: var(--border);
+  border: none;
+  border-radius: 6px;
+}
+
+/* Progress bar (front) */
+::-webkit-progress-value {
+  background-color: var(--focus);
+  border: none;
+  border-radius: 6px;
+}
+
+::-moz-progress-bar {
+  background-color: var(--focus);
+  border: none;
+  border-radius: 6px;
+}
+
+/* Chrome */
+progress:not([value])::before {
+  content: ' ';
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  background-image:
+    linear-gradient(
+      -60deg,
+      transparent 33%,
+      rgba(0, 0, 0, 0.4) 33%,
+      rgba(0, 0, 0, 0.4) 66%,
+      transparent 66%
+    );
+  animation: stripes 3s linear infinite;
+  border-radius: 6px;
+  background-size: 35px 20px, 100% 100%, 100% 100%;
+  background-color: var(--border);
+}
+
+/* Safari */
+progress:not([value])::-webkit-progress-bar {
+  background-image:
+    linear-gradient(
+      -60deg,
+      transparent 33%,
+      rgba(0, 0, 0, 0.4) 33%,
+      rgba(0, 0, 0, 0.4) 66%,
+      transparent 66%
+    );
+  animation: stripes 3s linear infinite;
+  border-radius: 6px;
+  background-size: 35px 20px, 100% 100%, 100% 100%;
+  background-color: var(--border);
+}
+
+/* Mozilla */
+progress:not([value])::-moz-progress-bar {
+  background-image:
+    linear-gradient(
+      60deg,
+      transparent 33%,
+      rgba(0, 0, 0, 0.4) 33%,
+      rgba(0, 0, 0, 0.4) 66%,
+      transparent 66%
+    );
+  animation: stripes 3s linear infinite;
+  border-radius: 6px;
+  background-size: 35px 20px, 100% 100%, 100% 100%;
+  background-color: var(--border);
+}
```

Comment:
I love the progress styles and the new meter styles! Great job with everything. It looks like there's a small inconsistency, there appears to be a bit of top space above the fill for progress bars which shouldn't be there. That isn't an issue with the meters.

Progress:

![](https://doggo.ninja/n8jRLW.png)

Meter:

![](https://doggo.ninja/EuO10z.png)

Reply:
Could you give more information about this? I'm not able to reproduce it on my end. However, I do have to say that there _is_ a slight border on determinate progress but it's coming even for me.
<img width="1120" alt="Screenshot 2021-09-01 at 10 19 11 PM" src="https://user-images.githubusercontent.com/43945468/131711425-9ac761e5-5a9e-4bb8-a1c7-12c26ef3b35c.png">

Reply:
I'm able to reproduce it on Brave (Chromium) on Linux, but not Firefox.

Reply:
I tried with Brave on macOS and it’s being rendered properly. I don’t have a Linux system so not able to test in that. How should I proceed from here?
<img width="1552" alt="Screenshot 2021-09-03 at 12 16 17 AM" src="https://user-images.githubusercontent.com/43945468/131899668-aa68b228-44e8-4558-90a9-5e152b9cba0c.png">

Reply:
That's super odd, I'll check it out myself and see what I can do.

Reply:
Is this issue still open?

Reply:
It is! I unfortunately haven't had that much time recently so I'm a bit behind, but feel free to take a look.